Docker/test/Dockerfile

```diff
@@ -5,7 +5,6 @@ ENV GO111MODULE=on
 COPY go.mod .
 COPY go.sum .
 RUN go mod download
-ADD docker-run-tests.sh /root/run-tests.sh
+ADD ./Docker/test/docker-run-tests.sh /root/run-tests.sh
 COPY . .
-ADD config-docker.toml /etc/efes.toml
```

Comment:
Is it okay to run tests without a config?

Reply:
Tests will fail in that case. The reason why I removed this line is because we share config for `efes-tracker` and `test` container in tests. (and additionally `efes-server`)
I wanted all those to share config, only solution made sense for me was putting the config into the container through a volume. 

I am open to anything else, if you have any other ideas please share.

Reply:
I'd rather put config into images rather that using volume mount.

Reply:
We now have config files for each image folder and they are included in the image.